test/ui-automation/test/helpers/wallet.js

```diff
@@ -213,6 +221,23 @@ async function _changeLocale() {
     });
 }
 
+async function _changeVaultLocale() {
+    const localeSwitcherLink = await $("a*=Français");
+    await localeSwitcherLink.waitForExist();
+    await localeSwitcherLink.click();
+    // Verifying lang attribute is set to French on the page
+    expect(browser.config.walletURL).toHaveText('fr');
```

Comment:
'fr' can be a part of the page's url, we shouldn't test it this way. We should probably use https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language or make sure that 'fr' appears right after the base of the `walletUrl`

Reply:
updated using `expectChai(await browser.getUrl()).to.include('/fr/');`